src/app/carbon-estimator-form/carbon-estimator-form.component.spec.ts

```diff
@@ -18,7 +32,6 @@ describe('CarbonEstimatorFormComponent', () => {
 
   it('should create component form in a valid state', () => {
     expect(component).toBeTruthy();
-    component.ngOnInit();
```

Comment:
I don't think we need these explicit calls because `ngOnInit` is being called in the `beforeEach` block.